app/models/party_guest.rb

```diff
@@ -0,0 +1,4 @@
+class PartyGuest < ApplicationRecord
+  belongs_to :user
+  belongs_to :viewing_party
+end
```

Comment:
I added a `  validates :host, inclusion: { in: [true, false] }` on my PR that we may or may not want to use.